app/views/finders/_hidden_clearable_facet.html.erb

```diff
@@ -1,12 +1,12 @@
-<div id="<%= hidden_clearable_facet.key %>" style="display: none">
```

Comment:
Wow! I did not know we had hidden fields implemented this way. Great to get them removed 👍

Reply:
I know, I was quite surprised too!